docs/make.jl

```diff
@@ -17,7 +17,7 @@ if Base.active_project() != joinpath(@__DIR__, "Project.toml")
 end
 
 # (b) Did someone say render? Then we render!
-if "--quarto" ∈ ARGS
+if true || "--quarto" ∈ ARGS
```

Comment:
What does this approve? This means quarto is always run.

If you want to run this locally, the make.jl is executable and on terminal you can just use `docs/make.jl --quarto`.

Reply:
Sorry, I just meant to use it locally. I run basically everything through VSCode's "execute file in REPL" command so it's easier to just add a hack like this.

Reply:
ah ok, sure if I would know a nicer way we could add that as well, for now this allows to disable quarto un by default and only activate it if necessary. You can also just call `quarto render file.qmd` or even the quarto render shortcut in VS Code.